examples/vision/ipynb/image_classification_efficientnet_fine_tuning.ipynb

```diff
@@ -1,733 +1,766 @@
 {
```

Comment:
I'm surprising to see a complete rewrite of the notebook file -- it should just be a 2 line change?

Reply:
Same here :o 

Could it be because I edited it in Colab and committed directly from there?

Reply:
Most likely. You can either apply the edit to the ipynb by hand or regenerate the ipynb with the `tutobook.py` function `py_to_nb` (that way you don't have to execute the notebook, which takes time)

Reply:
Okay. Then I will work on the JSON of the ipynb directly so that the state does not get changed.

Reply:
@fchollet fixed it up. 

Operating directly on the JSON of the notebook seems like an easier choice for small changes like this.